src/agri_ai/line_bot/webhook.py

```
@@ -72,74 +50,118 @@ async def health_check():
-        except InvalidSignatureError:
-            raise HTTPException(status_code=400, detail="Invalid signature")
-        except LineBotApiError as e:
-            logger.error(f"LINE Bot API エラー: {e}")
-            raise HTTPException(status_code=500, detail="LINE Bot API error")
-        
-        return {"status": "ok"}
-        
+        handler.handle(body.decode("utf-8"), signature)
+    except InvalidSignatureError:
+        logger.error("署名検証に失敗しました。")
+        raise HTTPException(status_code=400, detail="Invalid signature")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)